src/app/(app)/(tenants)/tenants/[slug]/(home)/products/[productId]/page.tsx

```
@@ -0,0 +1,30 @@
+import { Suspense } from "react";
```

ESLint

[error] 'Suspense' is defined but never used.

(@typescript-eslint/no-unused-vars)